backend/putio/object.go

```diff
@@ -248,7 +248,10 @@ func (o *Object) Open(ctx context.Context, options ...fs.OpenOption) (in io.Read
 		_ = resp.Body.Close()
 		return nil, fserrors.NoRetryError(err)
 	}
-	return resp.Body, err
```

Comment:
I'm not a big fan of bare returns with unamed parameters...

How about this which would be a bit more traditional?

```go
if err != nil {
    return nil, err
}
return resp.Body, nil
```

Reply:
I have updated the commit.